### PR TITLE
Build dish endpoint and it's schema #11

### LINKED ISSUE #11: Build dish endpoint and it's schema

### PR DESCRIPTION
**Pull request by TesnimNakilci**
Fixes #11

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -9,6 +10,6 @@ const port = process.env.NODE_LOCAL_PORT;
 app.use('/api', apiRoutes);
 
 app.listen(port, () => {
-  console.log(`Server listening on port ${port}`);
+  // console.log(`Server listening on port ${port}`);
```
Comment (motaz99) [inline src/app.js]: Why did you comment this line?

File: src/db.js [hunk 2]
```diff
@@ -8,11 +8,11 @@ const connectToMongo = () => {
   const db = mongoose.connection;
 
   db.once('open', () => {
-    console.log('Database connected: ', url);
+    //    console.log('Database connected: ', url);
   });
 
   db.on('error', (err) => {
-    console.error('Database connection error: ', err);
+    // console.error('Database connection error: ', err);
```
Comment (motaz99) [inline src/db.js]: why are you commenting the console.log?

File: src/models/dish.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    required: true,
+  },
+  /* picture: {
+    type: String,
+    required: true,
+  }, */
+  rating: String,
+});
+
+// Create a Dish model from the schema
```
Comment (motaz99) [inline src/models/dish.js]: We don't need this comment

File: src/models/dish.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+// Define the Dish schema
```
Comment (motaz99) [inline src/models/dish.js]: We don't need this comment

File: src/routes/dish.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+
+const router = express.Router();
+const dishController = require('../controllers/dishController');
+
+router.use(express.json());
+
+router.post('/', dishController.createDish); // Create a route for creating a new dish
+router.get('/', dishController.getAllDishes); // Create a route for getting all dishes
```
Comment (motaz99) [inline src/routes/dish.js]: We don't need those comments

File: yarn.lock
Comment (motaz99) [inline yarn.lock:1]: Why did you delete this file?

File: package.json [hunk 6]
```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "dotenv": "^14.3.2",
-    "express": "^4.17.2",
+    "express": "^4.18.2",
```
Comment (motaz99) [inline package.json]: why did you change the version of express?

**Review by motaz99 — CHANGES_REQUESTED** [review]

File: src/app.js (hunk 1 above)
Comment (TesnimNakilci) [reply src/app.js]: cause it gave me a warning  when ı pushed it

File: src/db.js (hunk 2 above)
Comment (TesnimNakilci) [reply src/db.js]: cause gave me a warning  when ı pushed it like these: 
11:5  warning  Unexpected console statement  no-console
 15:5  warning  Unexpected console statement  no-console

File: src/models/dish.js (hunk 3 above)
Comment (TesnimNakilci) [reply src/models/dish.js:25]: ok, I can delete it.

File: src/routes/dish.js (hunk 5 above)
Comment (TesnimNakilci) [reply src/routes/dish.js:9]: ok, I can delete it.

File: package.json (hunk 6 above)
Comment (TesnimNakilci) [reply package.json]: I didn't change the version of express.

File: yarn.lock
Comment (TesnimNakilci) [reply yarn.lock:1]: cause ı had error after change node and npm version

File: src/models/dish.js (hunk 3 above)
Comment (TesnimNakilci) [reply src/models/dish.js]: This comment is for illustration only

File: src/models/dish.js (hunk 4 above)
Comment (TesnimNakilci) [reply src/models/dish.js]: This comment is for illustration only

File: src/routes/dish.js (hunk 5 above)
Comment (TesnimNakilci) [reply src/routes/dish.js]: These comments are for illustration only

File: src/app.js (hunk 1 above)
Comment (motaz99) [reply src/app.js]: It shouldn't give any warning, I use them on my side and they didn't give any error.

File: src/db.js (hunk 2 above)
Comment (motaz99) [reply src/db.js]: They shouldn't it must be deferent issue.

File: src/models/dish.js (hunk 3 above)
Comment (motaz99) [reply src/models/dish.js]: The code can explain itself.

File: src/models/dish.js (hunk 4 above)
Comment (motaz99) [reply src/models/dish.js]: The code can explain itself.

File: src/routes/dish.js (hunk 5 above)
Comment (motaz99) [reply src/routes/dish.js]: The code can explain itself.

File: package.json (hunk 6 above)
Comment (motaz99) [reply package.json]: Yes you did, you might don't know how did you do it but you did. because it shows into your PR that you changed the version from 4.17.2 to 4.18.2
